includes/functions.php

```diff
@@ -2013,7 +2032,6 @@ function tsml_scan_data_source($data_source_url)
 			if (array_key_exists($data_source_url, $tsml_data_sources)) {
```

Comment:
Hey I know this is outside the review, but on my fresh install with no $tsml_notification_addresses, this import logic gets totally skipped (line 2031 `!empty($tsml_notification_addresses)`). Is this intended? Seems like if you have no notification address, the scan doesn't import.